frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -342,18 +249,89 @@ export const PrometheusMultilineUtilizationItem = withDashboardResources<
   },
 );
 
-export const UtilizationCard = () => {
+const UtilizationCardNodeFilter: React.FC<UtilizationCardNodeFilterProps> = ({
+  machineConfigPools,
+  onNodeSelect,
+  selectedNodes,
+}) => {
   const { t } = useTranslation();
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
```

Comment:
I don't mind being explicit, just a note that TypeScript can infer it's a boolean if you pass an initial value to `useState` without needing `<boolean>`.